test/functional/terminal/scrollback_spec.lua

```diff
@@ -445,18 +436,14 @@ describe("'scrollback' option", function()
     -- Terminal job data is received asynchronously, may happen before the
     -- 'scrollback' option is synchronized with the internal sb_buffer.
     command('sleep 100m')
-    if iswin() then
-      feed_data('for /L %I in (1,1,40) do @(echo line%I)\r')
-    else
-      feed_data('awk "BEGIN{for(n=1;n<=40;n++) print \\\"line\\\" n}"\n')
-    end
 
-    screen:expect{any='line40                        '}
+    feed_data(nvim_dir.."/shell-test REP 41 line"..(iswin() and '\r' or '\n'))
+    screen:expect{any='40: line                      '}
 
     retry(nil, nil, function() expect_lines(58) end)
     -- Verify off-screen state
-    eq((iswin() and 'line36' or 'line35'), eval("getline(line('w0') - 1)"))
-    eq((iswin() and 'line27' or 'line26'), eval("getline(line('w0') - 10)"))
+    eq('35: line', eval("getline(line('w0') - 1)"))
+    eq('26: line', eval("getline(line('w0') - 10)"))
```

Comment:
the ternary is still needed for windows:

```
[  FAILED  ] test/functional\terminal\scrollback_spec.lua @ 413: 'scrollback' option deletes lines (only) if necessary
test/functional\terminal\scrollback_spec.lua:445: Expected objects to be the same.
Passed in:
(string) '36: line'
Expected:
(string) '35: line'
stack traceback:
	test/functional\terminal\scrollback_spec.lua:445: in function <test/functional\terminal\scrollback_spec.lua:413>
```

Reply:
Fixed.